Tests/Test_BSP_Timer.c

```diff
@@ -0,0 +1,34 @@
+#include "BSP_Timer.h"
+
+/** 
+ * @brief   Test function to see if the first Timer works
+ * @param   None
+ * @return  none (will print a string on console)
+ */  
+void functionOne()
+{
+  printf("First Function worked\n");
+
+}
+
+/** 
+ * @brief   Test function to see of the second Timer works 
+ * @param   None
+ * @return  none (will print a string on console)
+ */  
+void functionTwo()
+{
+  printf("Second Function worked\n");
+}
+
+
+int main()
+{
+
+    BSP_Timer_Init(1000,functionOne,0);
+    BSP_Timer_Init(100,functionTwo,1);
```

Comment:
The point of creating an enum is to be able to call functions using the names given to them, not the number value.